src/sas/qtgui/Utilities/Preferences/PreferencesWidget.py

```diff
@@ -1,6 +1,6 @@
 import logging
 
-from PyQt5.QtGui import QIntValidator, QDoubleValidator
+from PyQt5.QtGui import QIntValidator, QDoubleValidator, QValidator
```

Comment:
QValidator syntax changed in Qt6, please use care when merging, depending on whether the pyside6 branch is merged first.

Reply:
Good point. This PR should only be merged once #2478 is finalized (I'll be testing it today) and this branch is brought up to date with that change.